aqua/input-leap/Portfile

```
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        input-leap input-leap 198779cf9e798aac6ad41427e32d031305736de3
+version             20230416
+revision            0
+categories          aqua net sysutils
+platforms           darwin
+license             GPL-2
+maintainers         iki.fi:koston
```

Please add your GitHub handle:
```suggestion
maintainers         {iki.fi:koston @Koston-0xDEADBEEF}
```
Unless you have strong objections, please add `openmaintainer` which communicates that others may make minor changes to the port without your prior approval:
```suggestion
maintainers         {iki.fi:koston @Koston-0xDEADBEEF} openmaintainer
```